spec/lib/solidus_subscriptions/subscription_generator_spec.rb

```diff
@@ -44,6 +44,15 @@
         payment_source: payment_source,
       )
     end
+
+    it 'cleanups the subscription line items fields duplicated on the subscription' do
+      subscription_line_item = create(:subscription_line_item)
+
+      described_class.activate([subscription_line_item])
+
+      attrs = %i[interval_length interval_units end_date]
+      expect(subscription_line_item.reload.slice(attrs).values).to eq [nil, nil, nil]
```

Comment:
What do you think of using `have_attributes`? I think it will be easier to read and maintain.

It could be something like:
```suggestion
      expect(subscription_line_item.reload).to have_attributes(
        interval_length: nil,
        interval_units: nil,
        end_date: nil,
      )
```

Reply:
Sure, it looks like a nice improvement: https://github.com/solidusio-contrib/solidus_subscriptions/pull/202/commits/11fa55f3c6a6a6b95314d1967f354f48b3c17416